scipy/stats/tests/test_qmc.py

```diff
@@ -16,6 +16,9 @@
                               _perturb_discrepancy)  # noqa
 
 
+generic_rng = np.random.default_rng()
```

Comment:
Doe this get the same seed as the docstrings? 
It has always been recommended to me to use a seed in tests. Non-reproducible errors that are seed dependent are not very helpful.

Reply:
Right I can change this. Here it was for calls which are not supposed to really matter and avoid re-writing this. I will do a fixture maybe. In the end cases like these could be a good fit for hypothesis. I will keep an eye on what you do here and copy 😉

Reply:
In tests, it really pays to control the seed even if the data is flowing through something that's supposed to handle every possible output. 6-months-from-now-you will thank you when he has to debug something unrelated and doesn't have to worry about the data flowing through changing each time.

Reply:
Done

Reply:
What we're saying is that _all_ tests should be repeatable. There are several still for which seed is (still) removed. I didn't comment on all of them.